src/survey-enfr.json

```diff
@@ -11,7 +11,7 @@
+                        "default": "<h3>1. What is the Algorithmic Impact Assessment?</h3>The AIA is a questionnaire designed to help you assess and mitigate the impacts associated with deploying an automated decision system. The AIA also helps identify the impact level of your automated decision system under the Directive on Automated Decision-Making. The questions are focused on your business processes, your data, and your system design decisions.<div style='margin-left: 55px;'><h3>a. How does it work?</h3>The questionnaire asks you around 60 questions related to your business process, data and system designed decisions. The results will demonstrate your impact level as well as a link where you can learn about what your requirements are under the Directive. The questionnaire takes at least 35 minutes to complete.<h3>b. Where can I find out why my system was rated a certain level?</h3><p> When you complete the questionnaire, a detailed results page will be displayed. You can print it or save it as PDF.</p></div><h3>2. Can I save my responses and complete the questionnaire later?</h3><p>Yes. When you want to save your progress, click on the 'Save' button at the top of the window and save the JSON file (machine readable structure text document) to your local folders. If you want to continue where you left off, click on the 'Choose file' button and select your saved JSON file.</p> <p>All data will be deleted if you click on the button 'Start over'</p> <div style='margin-left: 55px;'> <h3>a. Do you store my answers?</h3><p>No. We don't store your responses and data only remains within your browser's local storage. Feel free to use this tool as many times as you need along the development of your system.</p></div><h3>3. Can I use this project for my government or company?</h3>Yes, this is an open source project released under the MIT licence and you are welcome to use it for your own needs. See the link above for the project repository. <br> <br>Please provide notice of licence and attribution of the work where required.<h3> 4. 'This is a work in progress and changes are being made on a daily basis to the application'. You can follow its development and even participate by joining us on GitHub at the link above. </h3> </br>If you find a bug or if you encounter a problem, please create an <a href='https://github.com/canada-ca/aia-eia-js/issues' target='_blank'>issue on the project site.</a><br>",
```

Comment:
It got rid of my changes for task #376 so I had to re-commit it and include it with task #368 .